src/api/types/index.tsx

```diff
@@ -86,6 +86,12 @@ export type FamilyStudentRequest = FamilyRequest & {
   parent: StudentRequest;
 };
 
+export type FamilySessionRequest = {
+  family: FamilyStudentRequest;
+  session: SessionListResponse["id"] | null;
+  preferred_class: ClassListResponse["id"] | null;
```

Comment:
nit: we don't really need to rely on the response of other API endpoints for this, so we can do `number | null`